project-communications/newsletter/newsletter-June21.md

```diff
@@ -3,50 +3,50 @@
 **Hi everyone!!**
 
 Firstly, thank you for opening and reading this month's newsletter 😄 We are so happy you are here!
+June is very quickly coming to an end and there has been a lot going on here at AutSPACEs. 
 
-June is showing us some warmer weather that I really hope is raising spirits! 
+There is a lot of change happening right now with the COVID roadmap having come out a couple of months ago and things starting to open up.
```

Comment:
If your participants include also those not from the UK, maybe drop the Covid roadmap or city opening up.